prisma/dev-seeds/user.ts

```diff
@@ -201,6 +201,17 @@ const users: User[] = [
         createdAt: new Date(),
         updatedAt: null,
     },
+    {
+        id: 10018,
+        firstName: "Greg",
+        lastName: "Maxin",
+        email: "gmaxin@uwaterloo.ca",
+        emailVerified: new Date(),
+        role: roles.PROGRAM_ADMIN,
+        image: null,
+        createdAt: new Date(),
+        updatedAt: null,
+    },
```

Comment:
@GodGreg  if you wanna persist this, can you also add the corresponding program_admin record? Otherwise I suggest deleting this and overriding existing user emails to seed when you want to test.